homeassistant/components/verisure/switch.py

```diff
@@ -77,16 +79,21 @@ def available(self) -> bool:
             and self.serial_number in self.coordinator.data["smart_plugs"]
         )
 
-    def turn_on(self, **kwargs: Any) -> None:
-        """Set smartplug status on."""
-        self.coordinator.verisure.set_smartplug_state(self.serial_number, True)
-        self._state = True
-        self._change_timestamp = monotonic()
-        self.schedule_update_ha_state()
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        """Turn the smartplug on."""
+        await self.async_set_plug_state(True)
 
-    def turn_off(self, **kwargs: Any) -> None:
-        """Set smartplug status off."""
-        self.coordinator.verisure.set_smartplug_state(self.serial_number, False)
-        self._state = False
-        self._change_timestamp = monotonic()
-        self.schedule_update_ha_state()
+    async def async_turn_off(self, **kwargs: Any) -> None:
+        """Turn the smartplug off."""
+        await self.async_set_plug_state(False)
+
+    async def async_set_plug_state(self, state: bool) -> None:
+        """Set smartplug state."""
+        command: dict[str, str | dict[str, str]] = (
+            self.coordinator.verisure.set_smartplug(self.serial_number, state)
+        )
```

Comment:
CI disagrees about how to split this line.

* https://github.com/home-assistant/core/actions/runs/4491557553/jobs/7916537447?pr=89318

The following change should resolve the the issue:

```suggestion
        command: dict[
            str, str | dict[str, str]
        ] = self.coordinator.verisure.set_smartplug(self.serial_number, state)
```